### PR TITLE
Reformatted 10 recipes recheck

### PR DESCRIPTION
**Pull request by anirudhjain75**

### CODE REVIEW

File: haiku-apps/moe/moe-1.1.2.recipe [hunk 1]
```diff
@@ -1,14 +1,13 @@
 SUMMARY="A cute mascot on the active window"
-DESCRIPTION="\
-Moe is a program that displays a cute mascot on the active window. \
-How about mind cooler for your desktop?"
+DESCRIPTION="Moe is a program that displays a cute mascot on the active \ window. How about mind cooler for your desktop?"
```
Comment (humdingerb) [inline haiku-apps/moe/moe-1.1.2.recipe]: Missing line-break after "\".

File: haiku-apps/niue/niue-20140701.recipe [hunk 2]
```diff
@@ -1,12 +1,11 @@
 SUMMARY="An easy to use but powerful development environment"
-DESCRIPTION="Niue is an easy to use but powerful development environment for \
-Haiku. It supports project management, syntax highlighting, GUI builder and more."
-
+DESCRIPTION="Niue is an easy to use but powerful development environment \ for Haiku. It supports project management, syntax highlighting, GUI builder \ and more."
```
Comment (humdingerb) [inline haiku-apps/niue/niue-20140701.recipe]: Missing line-break after "\", twice.

File: haiku-apps/niue/niue-20140701.recipe [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
 HOMEPAGE="https://github.com/HaikuArchives/Niue"
-SOURCE_URI="git+https://github.com/HaikuArchives/Niue#6745dcca121799572d0b4c98f9a006b6a657aac2"
-REVISION="1"
-LICENSE="Niue"
 COPYRIGHT="2008 T. Lansbergen"
+LICENSE="Niue"
+REVISION="1"
+SOURCE_URI="https://github.com/HaikuArchives/Niue/archive/6745dcca121799572d0b4c98f9a006b6a657aac2.tar.gz"
+CHECK_SUM256="7d82a163ae62fb45980df058c7010a3baba54d7648b8cf6229a32601ebd92610"
```
Comment (humdingerb) [inline haiku-apps/niue/niue-20140701.recipe]: Should be CHECKSUM_SHA256. SOURCE_DIR is missing.

File: haiku-apps/organizer/organizer-0.1.recipe [hunk 4]
```diff
@@ -1,25 +1,28 @@
...
+DESCRIPTION="An organizer that helps you keep track on your \
+appointments, notes and stuff."
+HOMEPAGE="https://github.com/HaikuArchives/Organizer"
 COPYRIGHT="2013 Marius Stene"
 LICENSE="Public Domain"
-HOMEPAGE="https://github.com/HaikuArchives/Organizer"
-SOURCE_URI="git+https://github.com/HaikuArchives/Organizer#32196761f4"
 REVISION="3"
+SOURCE_URI="https://github.com/HaikuArchives/Organizer/archive/32196761f461afba90f240f328f0ffdc790f2af7.tar.gz"
+CHECK_SUM256="e624273178cdcdf92348f78c985f1154f9d340b89e9255b02b80bff99cb654b0"
```
Comment (humdingerb) [inline haiku-apps/organizer/organizer-0.1.recipe]: Should be CHECKSUM_SHA256

File: haiku-apps/organizer/organizer-0.1.recipe [hunk 5]
```diff
@@ -28,12 +31,8 @@ BUILD_PREREQUIRES="
 	cmd:make
 	"
 
-REQUIRES="
-	haiku
-	"
-
 USER_SETTINGS_FILES="
-	settings/fSub directory"
+	settings/fsub directory"
```
Comment (humdingerb) [inline haiku-apps/organizer/organizer-0.1.recipe]: This is changed from "settings/fSub directory". Deliberate or accident.

File: haiku-apps/remotecontrol/remotecontrol-1.1.recipe [hunk 6]
```diff
@@ -1,19 +1,14 @@
 SUMMARY="Remote control your Haiku machines with ease!"
-DESCRIPTION="
-With the Remote Control app, you can control another Haiku computer also \
-running the Remote Control app, no matter where it is. The app is able to just \
-view what's on the screen, or also control the computer. This package contains \
-the RCClient client and RCServer server. Both are to be run in the terminal.
-"
+DESCRIPTION="With the Remote Control app, you can control another Haiku \ computer also running the Remote Control app, no matter where it is. The \ app  is able to just view what's on the screen, or also control the computer. \ This  package contains the RCClient client and RCServer server. Both are to \ be run in the terminal. "
```
Comment (humdingerb) [inline haiku-apps/remotecontrol/remotecontrol-1.1.recipe]: Missing line-breaks after "\".

File: haiku-apps/remotecontrol/remotecontrol-1.1.recipe [hunk 7]
```diff
@@ -1,19 +1,14 @@
...
 HOMEPAGE="http://thomas.thiriez.free.fr/beos/RemoteControl/index.html"
-SOURCE_URI="git://github.com/HaikuArchives/RemoteControl#c59e89e3643e4693a75e5b5456596533d068cc29"
-REVISION="1"
-LICENSE="RemoteControl Unknown"
 COPYRIGHT="2001 Thomas Thiriez"
+LICENSE="RemoteControl Unknown"
+REVISION="1"
+SOURCE_URI="https://github.com/HaikuArchives/RemoteControl/archive/c59e89e3643e4693a75e5b5456596533d068cc29.tar.gz"
+CHECKSUM_SHA256="6b6649d9efaa7e12418e2f1ca12ced1a9221ab6a7be5dbc8827aefcb94182bf0"
```
Comment (humdingerb) [inline haiku-apps/remotecontrol/remotecontrol-1.1.recipe]: Missing SOURCE_DIR.

File: haiku-apps/seeker/seeker-1.0.recipe [hunk 8]
```diff
@@ -26,12 +24,11 @@ REQUIRES="
 BUILD_REQUIRES="
 	devel:libsanta
 	"
-
 BUILD_PREREQUIRES="
 	haiku_devel
 	makefile_engine
 	cmd:gcc
-    cmd:ld
+       cmd:ld
```
Comment (humdingerb) [inline haiku-apps/seeker/seeker-1.0.recipe]: One tab indention. No spaces.

File: haiku-apps/tolmach/tolmach-1.1.0.recipe [hunk 9]
```diff
@@ -1,14 +1,14 @@
...
 COPYRIGHT="1999 Ivan V. Murasko"
 LICENSE="GNU GPL v2"
-HOMEPAGE="https://gitorious.org/desknotes/tolmach"
-SOURCE_URI="git+http://github.com/haikuarchives/tolmach.git#5a67e9d38292d5dd482e116e8ee7aa605dee5398"
 REVISION="1"
+SOURCE_URI="https://github.com/haikuarchives/tolmach/archive/5a67e9d38292d5dd482e116e8ee7aa605dee5398.tar.gz"
+CHECKSUM_SHA256="44da1dd27ff36465be712fc21ace226e00cdd1acd314074ae473ab31642a3cff"
+SOURCE_DIR="Tolmach-5a67e9d38292d5dd482e116e8ee7aa605dee5398"
 
 ARCHITECTURES="x86_gcc2 x86 ?x86_64"
```
Comment (humdingerb) [inline haiku-apps/tolmach/tolmach-1.1.0.recipe]: No empty lines between PROVIDES and REQUIRES and BUILD_REQUIRES and BUILD_PREREQUIRES.

**Comment by waddlesplash** [comment]
Also please squash your commits.

**Comment by waddlesplash** [comment]
All recipes have been reformatted automagically now, unneeded

**Pull request closed without merging**